web/tsconfig.node.json

```diff
@@ -10,5 +10,5 @@
     "strict": true,
     "target": "ESNext"
   },
-  "include": ["vite.config.ts", "cypress.config.ts", "scripts", "geo"]
+  "include": ["vite.config.cts", "cypress.config.ts", "scripts", "geo"]
```

Comment:
Ugh, so annoying with all the CommonJS file extensions 😬

Reply:
We could probably convert it to a module and just use .ts but I didn't want to make more changes than absolutely necessary in this PR.